www/docs/deploying.md

```diff
@@ -32,7 +32,8 @@ Within that file, you should perform any necessary setup and then spin up the se
 ### Data Streaming
 
 As described in the documentation for [resources and services](resources.md#resource-http-api), the Skip reactive service HTTP interface exposes control operations on one port and data streaming on another; the default streaming port is `8080` and the default control port is `8081`.
-Your application should expose _only_ the streaming port to the outside world; using HAProxy we can restrict and direct traffic as follows:
+**Important**: your application should expose _only_ the streaming port to the outside world, the control port should be hidden.
```

Comment:
Maybe throw in an analogy with exposing your database connection to the outside world for clarity?

Reply:
How about now?

Reply:
LGTM!